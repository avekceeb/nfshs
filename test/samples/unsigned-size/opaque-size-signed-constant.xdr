/*
Only unsigned constants may be used as size specifications for
arrays. If an identifier is used, it must have been declared
previously as an unsigned constant in a "const" definition.
*/

const N = -1;
struct S1 { opaque data[N]; };