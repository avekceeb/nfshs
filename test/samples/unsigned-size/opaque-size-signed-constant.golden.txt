test/samples/unsigned-size/opaque-size-signed-constant.xdr:8:26:
Identifier "N" used as array size specification at test/samples/unsigned-size/opaque-size-signed-constant.xdr:8:25 must have been declared previously as an unsigned constant
